test/test_ea_mu_plus_lambda.py

```diff
@@ -221,3 +221,46 @@ def test_create_new_parent_population(population_params, genome_params, ea_param
     # we picked the first three individuals
     new_parents = ea._create_new_parent_population(3, pop.parents)
     assert new_parents == pop.parents[:3]
+
+
+def test_update_n_objective_calls(population_params, genome_params, ea_params):
+    def objective(individual):
+        individual.fitness = float(individual.idx)
+        return individual
+
+    # create a population and an instance of the evolutionary algorithm and
+    # check that number of objective calls is initially zero
+    pop = cgp.Population(**population_params, genome_params=genome_params)
+    ea = cgp.ea.MuPlusLambda(**ea_params)
+    assert ea.n_objective_calls == 0
+
+    # after initialization of parents number of objective calls must be equal
+    # to the number of parents
+    ea.initialize_fitness_parents(pop, objective)
+    assert ea.n_objective_calls == population_params["n_parents"]
+
+    # count the number of objective calls by adding the number of offspring with fitness = None
+    # at each generation
+    n_objective_calls = ea.n_objective_calls
+    n_generations = 100
+    for _ in range(n_generations):
+        offsprings = ea._create_new_offspring_generation(pop)
+        combined = offsprings + pop.parents
+        n_objective_calls += sum([1 for ind in combined if ind.fitness is None])
+        combined = ea._compute_fitness(combined, objective)
+        assert n_objective_calls == ea.n_objective_calls
+
+    # create a population with mutation rate one and check that the number
```

Comment:
nice idea for a second test! but i'd suggest to move it to a separate `test_` function with an appropriate name.